### PR TITLE
[MM-44428]: Advanced text editor image pasting and drag drop issue

### PR DESCRIPTION
**Pull request by AshishDhama**
#### Summary
Fixes: Advanced text editor image pasting and drag drop issue

#### Ticket Link

  Fixes https://mattermost.atlassian.net/browse/MM-44428

#### Related Pull Requests
NONE

#### Release Note
```release-note
* Advanced text editor bugs
```

### CODE REVIEW

File: packages/mattermost-redux/src/selectors/entities/preferences.ts [hunk 1]
```diff
@@ -220,5 +220,5 @@ export function cloudFreeEnabled(state: GlobalState): boolean {
 }
 
 export function getIsAdvancesTextEditorEnabled(state: GlobalState): boolean {
-    return getFeatureFlagValue(state, 'AdvancedTextEditor') === 'true';
+    return getFeatureFlagValue(state, 'AdvancedTextEditor') === 'true' || true;
```
Comment (michelengelen) [inline packages/mattermost-redux/src/selectors/entities/preferences.ts:223]: I suppose this change was unintended.

**Review by michelengelen — APPROVED** [review]

**Review by Willyfrog — APPROVED** [review]

**Review by anurag6713 — APPROVED** [review]

**Comment by jgilliam17** [comment]
Thanks @AshishDhama 
Tested, looks good to merge.
- Verified thread drag & drop and paste file - working as expected.
- [E2E report](https://mattermost-cypress-report.s3.amazonaws.com/196648-dad2d7a-onprem-ent-webapp-pr-10336-mattermost/mm-ee-testdad2d7a/mochawesome.html)

**Pull request merged**